packages/suite/src/components/suite/Timer.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import formatDuration from 'date-fns/formatDuration';
+import { useLocales } from '@suite-hooks/useLocales';
+
+const formatTimeLeft = (deadline: Date, locale?: Locale) => {
+    const milisecondsLeft = new Date(deadline).getTime() - Date.now();
+    const timeoutDate = new Date();
+
+    timeoutDate.setHours(0, 0, 0, milisecondsLeft > 0 ? milisecondsLeft : 0);
+
+    const duration = {
+        hours: timeoutDate.getHours(),
+        minutes: timeoutDate.getMinutes(),
+    };
+
+    const formattedTimeLeft = formatDuration(duration, { locale })
+        .split(' ')
+        .map(item => {
+            if (Number.isInteger(item)) {
+                return item;
+            }
+
+            return item[0]; // Might not be relevant for all languages
```

Comment:
Any ideas how to handle formatting in the same style? (1h41min) Or we can pick a different style.

Reply:
I think this is not relevant to any language as it maps for example `['59', 'minutes']` to `5m` 🤔

Reply:
I would stick with whole words like `1 hour 59 minutes` or use [intlFormatDistance](https://date-fns.org/v2.29.3/docs/intlFormatDistance)  instead.

Reply:
[intlFormatDistance](https://date-fns.org/v2.29.3/docs/intlFormatDistance) doesn't have hours + minutes :( 

553294c65ebc0f98cdd168c80c14542ad398e998

Reply:
I think, that `in 2 hours` is better than `1 hour 59 minutes` since we are only guessing those estimates and they will be very inaccurate anyway... But let's leave it with hours and minutes for now 👌

Reply:
Yes, ideally, I would round to 15 min.

Reply:
Let's keep if for now and discuss the whole estimate thing once again, since it never was finalised. The best thing we have is the 2.5h per round estimation from Hynek.

Reply:
This estimate is assumed with random skipping of rounds (participate in 1-5 rounds). For the speedy setup we can assume 1 hour. And it all should take into account the max mining fee as well 😄 But we can ignore that for now..